### PR TITLE
added hello world program for lobster

### LINKED ISSUE #1080: Add Hello World in Lobster
If possible, please provide a link(s) to the language documentation to prove its existence:

- https://github.com/aardappel/lobster

If possible, please provide a code snippet of Hello World in the language:

```lang
print("Hello, World!")
````
source: https://htmlpreview.github.io/?https://raw.githubusercontent.com/aardappel/lobster/master/lobster/docs/command_line_usage.html

### PR DESCRIPTION
**Pull request by proabel**
I fixed #1080 Add Hello World in Lobster

### CODE REVIEW

**Comment by proabel** [comment]
How do i get it reviewed?

**Comment by proabel** [comment]
@hmajid2301  Deleted .DS_Store file.

File: archive/l/lobster/helloworld.lobster [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import std
+import vec
+import color
+
+fatal(gl_window("Shooter Tutorial", 640, 480))
```
Comment (jrg94) [inline archive/l/lobster/helloworld.lobster]: Does this launch a window with that title? If so, maybe we could try "Hello World" or something similar.

Comment (proabel) [reply archive/l/lobster/helloworld.lobster]: Yes I'll change the title.

**Comment by auroq** [comment]
Is this a graphical only language or are there command line options as well?
I'm wondering if there's a way we could fit this into our automated testing framework.

This seems like a great language for the [Game of Life project](https://sample-programs.therenegadecoder.com/projects/game-of-life/), but most projects are targeted as CLI programs. It'd be great to be able to run this like we do other languages.

**Comment by jrg94** [comment]
Good catch. Also, the README format should match the rest of the repo. That said, looks good.

**Comment by jrg94** [comment]
Merging this for now. We can chat about this again later.

**Pull request merged**